tests/contracts/randreceiver.cpp

```diff
@@ -41,38 +41,35 @@ CONTRACT randreceiver: public contract {
 
         print_f("receiverand called: assoc_id=%, random_value=%\n", assoc_id, rv_string );
 
-        set_last_result(assoc_id, random_value);
```

Comment:
You **mustn't remove random_value** in any place of this smart contract. This test smart contract implements a callback to receive a random value. The value is stored in a table in order to be validated from a unit test. Please **undo** all changes in this file.

Reply:
Please *undo* all the changes in this file.